tests/python/contrib/test_hexagon/test_launcher.py

```diff
@@ -304,11 +315,13 @@ def test_aot_executor(hexagon_session):
         lowered = tvm.relay.build(
             relay_mod,
             params=params,
-            target=tvm.target.Target(target_hexagon, host="c"),
+            target=tvm.target.Target(target_hexagon, host=aot_target_kind),
             runtime=Runtime("cpp"),
             executor=Executor("aot", {"unpacked-api": False, "interface-api": "packed"}),
         )
 
+    hexagon_session = hexagon_launcher.start_session(name=session_key)
+    hexagon_session.__enter__()
```

Comment:
why call this instead of with hexagon_session:?

Reply:
changed.